doc/rtd/explanation/format.rst

```diff
@@ -3,18 +3,65 @@
 User data formats
 *****************
 
-User data is opaque configuration data provided by a platform to an instance at
-launch configure the instance. User data can be one of the following types.
+User data is configuration data provided by a user of a cloud platform to an
+instance at launch. User data can be passed to cloud-init in any of many
+formats documented here. Each format will document a required header or
+content-type that must be present in the user data to be recognized as that
+format.
```

Comment:
"Each format will document a required header or content-type that must be present in the user data to be recognized as that format."
I found this a bit confusing - particularly "each format will document". Is the format doing the documenting? Or is it that each format requires a specific header or content type?

Reply:
I updated that sentence. Let me know if that's any better.

Reply:
It's a lot clearer to me :) thanks!